src/org/exist/xquery/value/DurationValue.java

```diff
@@ -211,18 +211,36 @@ protected Duration canonicalZeroDuration() {
 		return CANONICAL_ZERO_DURATION;
 	}
 	
-	public int getPart(int part) {
-		int r;
+	public int getPart(final int part) {
+		final int r;
 		switch(part) {
-			case YEAR: r = duration.getYears(); break;
-			case MONTH: r = duration.getMonths(); break;
-			case DAY: r = duration.getDays(); break;
-			case HOUR: r = duration.getHours(); break;
-			case MINUTE: r = duration.getMinutes(); break;
-            case SIGN: r = duration.getSign(); break;
+			case YEAR:
+				r = duration.getYears();
+				break;
+
+			case MONTH:
+				r = duration.getMonths();
+				break;
+
+			case DAY:
+				r = duration.getDays();
+				break;
+
+			case HOUR:
+				r = duration.getHours();
+				break;
+
+			case MINUTE:
+				r = duration.getMinutes();
+				break;
+
+			case SIGN:
+            	return duration.getSign();
```

Comment:
sorry I still don't understand.....
reformatting the code shows (hides?) at least there is a kind of interesting flow, which may or may not be intended. At least some comments helps to understand why things are done this way....